README.md

```diff
@@ -1,2 +1,52 @@
 # ft_transcendence
-🏓 Multiplayer pong tournaments
+🏓 Play pong online with your friends in real time
+
+# 🤝 Contributing
+
+If you want to contribute to this project you may follow a set of rules.
+
+### 1. Create a new issue or task:
+
+- With every contribution there must be a related issue/task, so if you
+don't have any, create a new one on the project board with some
+detailed info about what do you want to accomplish.
+
+### 2. Create a new branch:
+   
+- ⚠️ We **do not** push to `master`, never. So you'll need to create a branch
+to work on it.
+- Try to start always from `master` and do not forget to bring
+all new changes, preferably with `git pull --rebase` so you don't corrupt
```

Comment:
question: This should be `git rebase main`, from the branch we create?  (second point, instead of `git pull --rebase`)

Reply:
If you `pull` with `--rebase` you'll use a `rebase` strategy rather than `merge` to add remote changes into local. This is useful, for example, if you started working in master forgetting to create a new branch. Your commits will placed at the top of the tip like if you already started working from what is in the remote (which has latest changes)

The overall idea is everything that gets integrated into master should be the starting point of our task